apps/css-workshop/pages/index.html

```diff
@@ -141,25 +131,25 @@ <h1>iTwinUI-CSS</h1>
     <img
```

Comment:
We should update the paragraph (L129) with some brief text that clearly explains that these html pages are only meant for demoing the CSS classes from `@itwin/itwinui-css`, and also link back to our main documentation site.

Reply:
Changed paragraph to point users to docs site.